src/styles/index.ts

```diff
@@ -179,10 +179,9 @@ const webViewStyles = (theme: ThemeColors) =>
 
             pre: {
                 ...baseCodeTagStyles(theme),
-                paddingTop: 12,
-                paddingBottom: 12,
-                paddingRight: 8,
-                paddingLeft: 8,
+                paddingVertical: 8,
+                paddingHorizontal: 12,
+                fontSize: 13,
```

Comment:
@Krishna2323 Can we define the numeral in variable and use it as fontSize. If you check other fontSize in the file, we are using variable instead of a numeral.

Reply:
NAB: But I would have preferred it as a variable.